packages/gas-fee-controller/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.1]
+### Changed
+- **BREAKING:** Bump `@metamask/network-controller` to 2.0.0 ([#995](https://github.com/MetaMask/controllers/pull/995))
```

Comment:
It looks like this is still listed as a patch version. Could you update this to a major version bump?

Reply:
Oops! Yes. Updated in [a8bad34](https://github.com/MetaMask/controllers/pull/999/commits/a8bad345861297610417e212addb3820bb92666c).